src/pages/publication/contributors_tab/components/SortableTable.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { FC } from 'react';
+import { SortableContainer, SortableElement } from 'react-sortable-hoc';
+
+import {
+  Checkbox,
+  FormControlLabel,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  TextField,
+  Button,
+} from '@material-ui/core';
+import { Contributor, emptyContributor } from '../../../../types/contributor.types';
+import { Field } from 'formik';
+import AddContributor from '../AddContributor';
+import { useTranslation } from 'react-i18next';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+import FormCardSubHeading from '../../../../components/FormCard/FormCardSubHeading';
+import { removeDuplicatesByScn } from '../../../../utils/helpers';
+
+interface SortableItemProps {
+  contributor: Contributor;
+  placement: number;
+  onDelete: (index: number) => void;
+}
+
+const SortableItem = SortableElement(({ contributor, placement, onDelete }: SortableItemProps) => {
+  const { t } = useTranslation();
+
+  const index = placement - 1;
+
+  return (
+    <TableRow tabIndex={0} key={contributor.systemControlNumber}>
+      <TableCell align="left">
+        <FormCardSubHeading>{contributor.name}</FormCardSubHeading>
+        <Field name={`contributors[${index}].corresponding`}>
+          {({ field }: any) => (
+            <FormControlLabel
+              control={<Checkbox checked={field.value} {...field} />}
+              label={t('publication:contributors.corresponding')}
+            />
+          )}
+        </Field>
+        <div>
+          {contributor.corresponding && (
+            <Field name={`contributors[${index}].email`}>
+              {({ field }: any) => <TextField variant="outlined" label={t('profile:email')} {...field} />}
+            </Field>
+          )}
+        </div>
+      </TableCell>
+      <TableCell align="left">
+        {contributor.institutions?.map(institution => (
+          <div key={`${institution.id}`}>{institution.name}</div>
+        ))}
+      </TableCell>
+      <TableCell align="right">
+        <div>
+          <div>{placement}</div>
+          <div>
+            <Button color="secondary" onClick={() => onDelete(index)}>
+              <DeleteIcon />
+              {t('common:remove')}
```

Comment:
her trengs samme margin som jeg brukte. Vi bør kanskje lage noen komponenter for det snart

Reply:
denne PRen er uten styling